### DISCUSSION TITLE
Storing of files outside of Docker container

### DISCUSSION THREAD

**Discussion by whocarez1** [snipe/snipe-it#11064] [General]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

Hi guys,

I a bit unsure how to report this issue, but I think the documentation is unclear whent it comes to docker containers and storage. The Dockerfile https://github.com/snipe/snipe-it/blob/master/Dockerfile 
itself creates symlinks in /var/www/html/storage to /var/lib/snipeit where the actual storing of files really happens. However, I want to use docker compose in order to mount the storage of files
outside of the docker container. Right now I noticed that uploaded images are missing after destroying the container. Mounting /var/www/html/storage has shown that uploaded model
images disappear when the container is destroyed. This is mounted as a docker volume.

The provided docker-compose.yaml in the repository: https://github.com/snipe/snipe-it/blob/master/docker-compose.yml mounts only /var/www/html/storage/logs. I do not think this is enough?
Reading the docker documentation https://snipe-it.readme.io/docs/docker here, I find both references to mounting /var/lib/snipeit and /var/www/html. Which directory is correct to mount as a docker volume,
 to make sure that images are properly stored outside of the container? I find this a somewhat confusing.

I have been testing with version 5.3.6 of snipeit on Ubuntu 20.04, but this seems to apply to the latest version too.

### Reproduction steps

Described as above, using docker-compose yml and the documentation from the repo

### Expected behavior

Making sure that uploaded images are stored peristently outside of the container

### Snipe-IT Version

5.3.6

### Operating System

Ubuntu 20.04

### Web Server

nginx as reverse proxy

### PHP Version

7.4

**Comment by welcome** [comment]
👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
